### ISSUE TITLE
Missing normal md syntax images when exporting to a PDF on a md file which contains both img tag and normal md images.

### ISSUE THREAD

**Issue by 1999FordFocus** [typora/typora-issues#5077]
**Might related issue**
#3046 

**Bug details**
<img width="671" alt="pdf" src="https://user-images.githubusercontent.com/9652173/154836847-aa16b24c-039d-4313-9d02-5b26a6f9f201.png">

**OS**
macOS Monterey

**Typora Version**
1.1.3

**Comment by abnerlee** [comment]
Could you provide a sample md file?

**Comment by 1999FordFocus** [comment]
**The md file is simple:**
```
# Test

#### normal md syntax image

![image-20220219223333782](/Users/UserName/Documents/MyPath/.assets/Image/image-20220219223333782.png)

#### img tag

<img src="/Users/UserName/Documents/MyPath/.assets/Image/image-20220219223333782.png" alt="image-20220219223333782" style="zoom: 50%;" />
```

**Repeated exporting does not help.**

The image setting in my Typora is "copy image to custom folder".

**Comment by abnerlee** [comment]
should be fixed in 1.1.5
